k8s/aat/common/ia/timed-event-service.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: timed-event-service
+  namespace: ia
+  annotations:
+    fluxcd.io/automated: "true"
+    repository.fluxcd.io/java: java.image
+    filter.fluxcd.io/java: glob:prod-*
+spec:
+  releaseName: ia-timed-event-service
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: timed-event-service
+    version: 0.0.6
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ia/timed-event-service:prod-e7d6fcac
+      ingressHost: ia-timed-event-service-aat.service.core-compute-aat.internal
```

Comment:
this should be in the chart with environment templating

Reply:
:+1: